tests/phpunit/quiz_settings_test.php

```diff
@@ -167,6 +168,39 @@ public function test_browser_exam_keys_validation_errors($bek, $expectederrorstr
         $this->assertContains($expectederrorstring, $errors);
     }
 
+    public function test_validate_sebconfigfile_success() {
+        $user = $this->getDataGenerator()->create_user();
+        $this->setUser($user);
+        $xml = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
```

Comment:
is it possible to use file_get_contents(__DIR__ . '/sample_data/simpleunencrypted.seb'); here?

Reply:
Correct, the unit tests could have fixtures which are loaded like that. Yet those are not real seb files themselves.